code/controllers/subsystem/persistent_paintings.dm

```diff
@@ -0,0 +1,203 @@
+#define PAINTINGS_DATA_FORMAT_VERSION 1
+/*
```

Comment:
is this comment block an example of a stored painting? i'd add a comment clarifying that